libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -96,6 +96,18 @@ void AP_AHRS_NavEKF::update(bool skip_ins_update)
         update_EKF3();
         update_EKF2();
     }
+
+    // update home location for GPS baro correction for DCM
+    Location EKF_home_loc;
+    if (_ekf_type == 2 && _ekf2_started && (EKF2.getOriginHgtMode() & (1<<2)) && EKF2.healthy()) {
+        EKF2.getOriginLLH(-1, EKF_home_loc);
```

Comment:
OriginLLH can drift as the EKF moves it (with a corresponding shift in the reported position such that externally no drift is visible), I'm not sure that applying this to the DCM is valid, as the position could be drifting while DCM's estimation of position isn't undergoing the same shift.

I'm a bit fuzzy though on if this is still a concern or not, I know there was discussion of removing the behavior. @priseborough can you clarify if this might be a problem or not?